Tools/LyTestTools/ly_test_tools/o3de/editor_test.py

```diff
@@ -805,16 +392,16 @@ def _exec_editor_test(self, request: _pytest.fixtures.FixtureRequest,
             editor.wait(test_spec.timeout)
             output = editor.get_output()
             return_code = editor.get_returncode()
-            editor_log_content = editor_utils.retrieve_editor_log_content(run_id, log_name, workspace)
+            editor_log_content = editor_utils.retrieve_test_log_content(run_id, LOG_NAME, workspace)
```

Comment:
Use the argument `log_name` of the function not the constant `LOG_NAME`. There are many cases in this file that need to be updated